src/services/importTracker.ts

```diff
@@ -577,22 +578,22 @@ namespace ts.FindAllReferences {
         }
     }
 
-    function getExportEqualsLocalSymbol(importedSymbol: Symbol, checker: TypeChecker): Symbol {
+    function getExportEqualsLocalSymbol(importedSymbol: Symbol, checker: TypeChecker): Symbol | undefined {
         if (importedSymbol.flags & SymbolFlags.Alias) {
-            return Debug.checkDefined(checker.getImmediateAliasedSymbol(importedSymbol));

```

Comment:
I'm not sure why those `Debug.checkDefined` calls were here originally, but it seems like a good change to remove them to avoid crashes. As far as I understand, at least in the `export = _` and `module.exports = _` cases, it makes no sense to expect a symbol to always be defined.